### PR TITLE
Added in :gamecount and updated :elitecount

### PR DESCRIPTION
**Pull request by RealJammy**
I felt we needed a way to see how many people have done a base, please give feedback on implementation on this!

I reused a lot of code from the `:elitecount` command.

Please review code owo

### CODE REVIEW

File: cdbot/constants.py [hunk 1]
```diff
@@ -140,7 +144,7 @@ class Exchange:
     "https://pbs.twimg.com/profile_images/921313066515615745/fLEl2Gfa_400x400.jpg"
 )
 ELITECOUNT_ENABLED = True
-
+GAMECOUNT_ENABLED = True
```
Comment (pwnker) [inline cdbot/constants.py:147]: Tbh we should do stuff like this with the database this should be changeable with a command. But its probs fine. Cd dead anyway

**Review by pwnker — COMMENTED** [review]
I've not tested this but it looks alright.

File: cdbot/constants.py [hunk 2]
```diff
@@ -87,8 +87,12 @@ class VET2020:
     class Exchange:
         SHORTLIST = int(environ.get("EXCH_S_MEMBERS_ID", "582894164597932034"))
         CONFIRMED = int(environ.get("EXCH_C_MEMBERS_ID", "585150522336608256"))
-
-
+    class Base:
+        HQ = int(environ.get("HQ_100_MEMBERS_ID", "640481696541704192"))
```
[the remark this answers is not in the document]
Comment (RealJammy) [reply cdbot/constants.py:91]: 100 as in 100% completion

[the remark this answers is not in the document]
Comment (RealJammy) [reply cdbot/constants.py:91]: doesnt really matter much, just easier for people to understand looking back at the code imo

File: cdbot/constants.py (hunk 1 above)
Comment (thebeanogamer) [reply cdbot/constants.py:147]: No we shouldn't do this with the database, none of CD bot is setup for feature flags to live in a database

File: cdbot/cogs/cyber.py [hunk 3]
```diff
@@ -460,6 +462,47 @@ async def elitecount(self, ctx: Context):
...
+        """
+        Gets the number of people who have done a base.
+        """
+        if GAMECOUNT_ENABLED:
+            basesections = {
+                "HQ": {"Jiaozi Slayers(completed HQ)": Roles.Base.HQ},
+                "MOON": {"Real Astronauts(completed moonbase)": Roles.Base.MOON},
+                "FORENSICS": {"Dedicated Strings Runners(completed forensics base)": Roles.Base.FORENSICS},
+                "VOLCANO": {"Professional Cyberists(completed volcano base)": Roles.Base.VOLCANO},
+                "FIELD MANUAL": {"Truly edited people(completed the field manual)": Roles.Base.FM},
```
[the remark this answers is not in the document]
Comment (thebeanogamer) [reply cdbot/cogs/cyber.py:476]: Also remove the FM role, we'll just get people asking us to give it and it's a joke.

**Review by thebeanogamer — CHANGES_REQUESTED** [review]
```
Command raised an exception: TypeError: unhashable type: 'dict'
```

**Review by Sh3llcod3 — CHANGES_REQUESTED** [review]
Please make the changes @thebeanogamer is suggesting.

File: cdbot/cogs/cyber.py [hunk 4]
```diff
@@ -460,6 +462,47 @@ async def elitecount(self, ctx: Context):
...
+                """
+            **Completion Statistics**
+            """
+            )
+
+            embed = Embed(
+                title=f"CyberStart Game Completion",
+                description=description,
+                colour=Colour(0x2596be),
+            )  # Cyber Start blue???? i don't really know the colour
```
[the remark this answers is not in the document]
Comment (RealJammy) [reply cdbot/cogs/cyber.py]: removed comment

File: cdbot/cogs/cyber.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (RealJammy) [reply cdbot/cogs/cyber.py]: Removed tag lines and FM role.

**Review by thebeanogamer — CHANGES_REQUESTED** [review]
Can you show me this working? It always crashes when I test it.

I'm going to move the Summer Camp into a different PR so it can be merged faster.

**Comment by RealJammy** [comment]
I'll just close the PR for now, and reopen when I fix it.

**Pull request closed without merging**